### PR TITLE
update to venture builder role

### PR DESCRIPTION
**Pull request by peterlangmar**
Tension: as I've started to work in this role with Feri, I've realized, how much different this role could become than we have imagined a year ago. Now, I'm closing a bit on the gap between theory and reality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,9 +182,12 @@ Accountabilities:
 - Facilitating idea pitches and hackathons
 - Helping the product team: to validate ideas, build MVP tests, make business model, find resources
 - Making spin-off business modell
+- Establishing acceleration / incubation program
+- Fundraising
 
 Domains:
 - Go-no-go of spin-offs through advising process
+- Business model and format of the venture builder / incubator / accelerator
```
Comment (cassus) [inline README.md]: What's the tension behind this restriction?

**Comment by peterlangmar** [comment]
There was an initial idea about the venture builder, which is getting more and more solid as we are diving into it. It is important, that this role has the domain to make changes of the original plans to adopt to the realities.

**Comment by cassus** [comment]
Let's talk about this
:exclamation:

**Comment by cassus** [comment]
I fixed the merge conflicts

**Pull request merged**